frontend/src/components/personal/achievements/Achievements.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react'
+import { Typography, Popover } from '@material-ui/core'
+import EmojiEventsOutlinedIcon from '@material-ui/icons/EmojiEventsOutlined'
+
+import { useStyles } from './Achievements.styles'
+
+export const AchievementsBoard = (): React.ReactElement => {
+    const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null)
```

Comment:
Why not just simply `useState`?
Better name for `anchorEl` is required. What is the purpose of this anchor element?

Reply:
It is ready material Ui component to show `achievement name` if hovered by mouse.